samples/vue/src/client.ts

```diff
@@ -2,7 +2,10 @@ import { Client } from '@umbraco/headless-client'
 
 const client = new Client({
   projectAlias: process.env.VUE_APP_UMBRACO__PROJECTALIAS!,
-  language: 'en-US'
+  apiKey: process.env.VUE_APP_UMBRACO__APIKEY,
+  language: 'en-US',
+  preview: true,
+  cdnProxyUrl: 'https://preview.rainbowsrock.net',
```

Comment:
I don't think we should leave anything pointing to our development environment in our public repos 🤔

Reply:
Not sure why I missed that one, only used if for testing

Reply:
It's removed now :)